internal/graphql/model/helpers.go

```diff
@@ -291,3 +291,31 @@ func SetOverrideValues(delete bool, adapterTracker models.AdaptersTrackerInterfa
 
 	return overrideValues
 }
+
+type K8sConnectionTracker struct {
+	mx              sync.Mutex
+	ContextToBroker map[string]string //ContextID -> BrokerURL
+}
+
+func (k *K8sConnectionTracker) Set(id string, url string) {
+	k.mx.Lock()
+	defer k.mx.Unlock()
+	k.ContextToBroker[id] = url
+}
+
+func (k *K8sConnectionTracker) Get(id string) (url string) {
+	k.mx.Lock()
+	defer k.mx.Unlock()
+	url = k.ContextToBroker[id]
+	return
+}
+
+func (k *K8sConnectionTracker) Log(l logger.Handler) {
```

Comment:
A couple of comments on these functions would be helpful.

Reply:
Sure